docs/GettingStarted.md

```diff
@@ -86,25 +86,20 @@ Open that file, and write the following code in it:
 <!--DOCUSAURUS_CODE_TABS-->
 <!--Sync Mode-->
 ```js
-const assert = require('assert')
-
 describe('webdriver.io page', () => {
     it('should have the right title', () => {
         browser.url('https://webdriver.io')
         const title = browser.getTitle()
-        assert.strictEqual(title, 'WebdriverIO · Next-gen browser automation test framework for Node.js')
+        expect(browser).toHaveTitle('WebdriverIO · Next-gen browser automation test framework for Node.js');
     })
 })
 ```
 <!--Async Mode-->
 ```js
-const assert = require('assert')
-
 describe('webdriver.io page', () => {
     it('should have the right title', async () => {
         await browser.url('https://webdriver.io')
-        const title = await browser.getTitle()
-        assert.strictEqual(title, 'WebdriverIO · Next-gen browser automation test framework for Node.js')
+        expect(browser).toHaveTitle('WebdriverIO · Next-gen browser automation test framework for Node.js');
```

Comment:
In async mode we need to await the expect call, afaik:

```suggestion
        await expect(browser).toHaveTitle('WebdriverIO · Next-gen browser automation test framework for Node.js');
```

Reply:
You are 100% right. Just tested this and the assertion doesn't fail if I don't await, so it's definitely needed.